CSS files/components/buttons/buttons.css

```diff
@@ -24,13 +26,130 @@
 
 }
 
+.btn-primary-outline{
+    background-color: var(--pure-white);
+    color: var(--primary-color);
+    border:1px solid var(--primary-color)
+}
+
+.btn-primary-outline:hover{
+    background-color: var(--primary-color);
+    color: var(--bg-color-white);
+}
+
+.btn-primary-disabled{
+    background-color: #FFA27F;
+    cursor: unset;
+}
+
+.btn-icon-primary{
+    font-size: 3.2rem;
+    color: var(--primary-color);
+    cursor: pointer;
+}
+.btn-icon-primary:hover{
+    color: var(--dark-primary);
+}
+
+.primary-link{
+    color: var(--primary-color);
+    text-decoration: underline 2px var(--primary-color);
+    font-size: 1.6rem;
+    cursor: pointer;
+}
```

Comment:
It would have better if you could have followed the same naming convention as you did for buttons: btn-primary. Similarly it would be link-primary.

Reply:
Yess